website/docs/guides/migration/versions/upgrading-to-dbt-utils-v1.0.md

```diff
@@ -0,0 +1,116 @@
+# dbt utils 1.0 Migration Guide
+
+For the first time, dbt utils is crossing the major version boundary. From [last month’s blog post](https://www.getdbt.com/blog/announcing-dbt-v1.3-and-utils/): 
+
+> It’s time to formalise what was already unofficial policy: you can rely on dbt utils in the same way as you do dbt Core, with stable interfaces and consistent and intuitive naming.
+
+Just like the switch to dbt Core 1.0 last year, there are some breaking changes as we standardised and prepared for the future. Most changes can be handled with find-and-replace; if you need help, post on the [Community Forum](https://discourse.getdbt.com) or in #package-ecosystem on Slack. 
```

Comment:
i think we use US spelling but am confirming this [here](https://dbt-labs.slack.com/archives/C03PUUV2GUD/p1669985818018669)
```suggestion
Just like the switch to dbt Core 1.0 last year, there are some breaking changes as we standardized and prepared for the future. Most changes can be handled with find-and-replace; if you need help, post on the [Community Forum](https://discourse.getdbt.com) or in #package-ecosystem on Slack. 
```

Reply:
- Changed to US spelling 🇳🇿🇳🇿🇳🇿🇳🇿
- It is intentionally past tense; the changes have already been made as we ~standardised~ standardized. 
- I don't like the bulletpoints for those two options, I've tried reworking it to be good as prose but let me know if you feel really strongly about this.